### ISSUE TITLE
Cannot build hypervisor with -j

### ISSUE THREAD

**Issue by rossburton** [projectacrn/acrn-hypervisor#2414]
There are race conditions in the makefile which means building with -j is impossible:

```
| DEBUG: Executing shell function do_compile
| NOTE: make -j 20 RELEASE=1 SYSROOT=/data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/recipe-sysroot O=/data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/build/ DESTDIR=/data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/image prefix=/usr bindir=/usr/bin libdir=/usr/lib datadir=/usr/share includedir=/usr/include systemd_unitdir=/lib/systemd HV_OBJDIR=/data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/build//hypervisor EFI_OBJDIR=/data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/build//efi-stub BOARD=nuc6cayh FIRMWARE=uefi -C hypervisor
| make: Entering directory '/data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/git/hypervisor'
| touch /data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/build//hypervisor/include/version.h
| ******* No ACPI info found *******
| Expected ACPI info header at bsp/include/nuc6cayh_acpi_info.h
|
| The ACPI info header for this board is not available. Please use
| an offline tool on the target board to generate a validated one.
| More details will be available soon.
| make: *** No rule to make target '/data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/build//hypervisor/release/librelease.a', needed by '/data/poky-tmp/master/work/intel_corei7_64-poky-linux/acrn-hypervisor/0.5-r0/build//hypervisor/acrn.out'.  Stop.
```

**Comment by miguelinux** [comment]
The fix is at #2404

**Comment by rossburton** [comment]
So it is thanks.
